.travis.yml

```diff
@@ -133,6 +133,15 @@ jobs:
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=address,integer,undefined CC=clang CXX=clang++"
 
+    - stage: test
+      name: 'x86_64 Linux  [GOAL: install]  [bionic]  [no depends, only system libs, sanitizers: address/leak (ASan + LSan) + undefined (UBSan) + integer] [unsigned char]'
+      env: >-
+        HOST=x86_64-unknown-linux-gnu
+        PACKAGES="clang llvm python3-zmq qtbase5-dev qttools5-dev-tools libssl1.0-dev libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev libprotobuf-dev protobuf-compiler libqrencode-dev"
+        NO_DEPENDS=1
+        GOAL="install"
+        BITCOIN_CONFIG="--enable-zmq --with-incompatible-bdb --with-gui=qt5 CFLAGS='-funsigned-char' CPPFLAGS='-DDEBUG_LOCKORDER -funsigned-char' --with-sanitizers=address,integer,undefined CC=clang CXX=clang++"
```

Comment:
Sorry to further nit-pick, but what are the advantages of making this a matrix job (duplicating the above and only changing this one)?

If there are none, I'd rather combine them.

Reply:
The advantage is that we get testing under the sanitizers for both signed and unsigned char builds.

Combining them would mean that we only get sanitizer testing for unsigned char builds.

Reply:
I guess the default is signed char, so developers should run those already.

If you insist, I'd rather combine it with one of the jobs further up (dash, or the bionic job right after the one using dash)

We already use 12 jobs and the free travis capacity is 5 jobs in parallel, so if we bloat the matrix further, travis might slow down some devs that run it on their own repo.

Reply:
Fixed!